lib/config/index.js

```diff
@@ -179,15 +179,13 @@ switch (config.imageUploadType) {
     config.allowedUploadMimeTypes = [
       'image/jpeg',
       'image/png',
-      'image/jpg',
       'image/gif'
     ]
     break
   default:
     config.allowedUploadMimeTypes = [
       'image/jpeg',
       'image/png',
-      'image/jpg',
       'image/gif',
       'image/svg+xml'
```

Comment:
Maybe we want to add the WebP image format as well? It's becoming more popular nowadays since Google's lighthouse recommends the usage of WebP format for web content.[[1]]

[1]: https://web.dev/uses-webp-images/

Reply:
I would not add anything new anymore as this is just a bug fix.

Reply:
okay